answerkey/streamingio/05.answer.md

```diff
@@ -5,13 +5,12 @@ extension [R](self: Pull[Int, R])
       window: collection.immutable.Queue[Int],
       p: Pull[Int, R]
     ): Pull[Double, R] =
-      p.uncons.flatMap {
+      p.uncons.flatMap:
         case Left(r) => Result(r)
         case Right((hd, tl)) =>
           val newWindow = if window.size < n then window :+ hd
                                              else window.tail :+ hd
           val meanOfNewWindow = newWindow.sum / newWindow.size.toDouble
           Output(meanOfNewWindow) >> go(newWindow, tl)
-      }
     go(collection.immutable.Queue.empty, self)
```

Comment:
To continue the definitions / paragraphs analogy, if there are no braces, perhaps we should have an empty line after end of definition.